server/routes/statistics.js

```diff
@@ -66,6 +69,12 @@ module.exports = function (config) {
 	getData(driver).done();
 
 	return assign({
+		'get-flow-data': function (query) {
+			return flowQueryHandler.resolve(query)(function (query) {
+				// data goes here
```

Comment:
I see utils secluded to external modules, but I don't see them used here to calculate needed periods (with which we will call `calculateStatusEventsSums` after #1758 is done)